scripts/unused-key-cleanup/cleanup-feedback-model.js

```diff
@@ -0,0 +1,32 @@
+/* eslint-disable */
+
+/*
+Delete unused keys for all formfeedback
+*/
+
```

Comment:
can i check when did we stop using the agency key in form feedback?

Reply:
https://github.com/datagovsg/formsg/pull/473